README.md

```diff
@@ -199,6 +200,80 @@ Although these arguments are optional, a (\*) indicates a recommended parameter
 | `hunkUrlTemplate`          | If provided, LaunchDarkly will attempt to generate links to your Git service provider per code reference. Example: `https://github.com/launchdarkly/ld-find-code-refs/blob/${sha}/${filePath}#L${lineNumber}`. Allowed template variables: `sha`, `filePath`, `lineNumber`. If `hunkUrlTemplate` is not provided, but `repoUrl` is provided and `repoType` is not custom, LaunchDarkly will automatically generate links to the repository for each code reference.      |                                |
 | `version`                  | If provided, the current `ld-find-code-refs` version number will be logged, and the scanner will exit with a return code of 0.                                                                                                                                                                                                                                                                                                                                           | `false`                        |
 
+### Configuring aliases
+
+Flag key aliases may be defined using a YAML file stored in your repository at `.launchdarkly/config.yaml`. Configuration types may be used in conjunction and defined more than once for comprehensive alias coverage.
```

Comment:
Not at the moment, mostly following conventions set by other tools that have configs like this. We could consider supporting having the `.ldignore` in there. 

Do you think we should change it? Since we're going for a prerelease first, I'm going to continue with how it is now pending further feedback

Reply:
seems like moving ldignore in here eventually would make sense